skimage/measure/_polygon.py

```diff
@@ -18,6 +19,12 @@ def approximate_polygon(coords, tolerance):
         Maximum distance from original points of polygon to approximated
         polygonal chain. If tolerance is 0, the original coordinate array
         is returned.
+    closed : boolean
+        If true, the algorithm assumes the given polygon is closed
+        (the first and last vertices are connected) and approximates using the
```

Comment:
true -> True

Describe "approximating using the first and last point" a bit more clearly.